### PR TITLE
homework2

### PR DESCRIPTION
**Pull request by aleksandrovruslan**
homework2
Screenshot_1605980230
Screenshot_1605980237
Screenshot_1605980246

### CODE REVIEW

File: app/src/main/res/layout/activity_movie_details.xml [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/margin_start_tagline_16"
+            android:layout_marginTop="@dimen/margin_top_tagline_4dp"
+            android:text="@string/tagline"
+            android:textColor="@color/colorRed"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/movie_name" />
+
+        <ImageView
```
Comment (alexander-smityuk) [inline app/src/main/res/layout/activity_movie_details.xml:81]: @aleksandrovruslan Please use RatingBar instead of 5 imageViews

**Review by alexander-smityuk — COMMENTED** [review]
Great job! Fix rating bar and will be approved.

File: app/src/main/res/layout/activity_movie_details.xml (hunk 1 above)
Comment (aleksandrovruslan) [reply app/src/main/res/layout/activity_movie_details.xml]: Fixed it.

**Review by alexander-smityuk — APPROVED** [review]
Greate! Approved 👍

**Pull request merged**